Gemfile

```diff
@@ -0,0 +1,7 @@
+# A sample Gemfile
```

Comment:
Add a readme! Write how far you got, what you wanted to do next. Write installation instructions and how to run tests too